src/main/java/Duke.java

```diff
@@ -1,10 +1,179 @@
+                    tasks.get(counter).print();
+                    counter++;
+                    System.out.println("    Now you have " + counter + " tasks in your list!");
+                    break;
+                }
+
+            case "deadline":
+                System.out.println("    _________________________________________");
+                System.out.println("    Please specify task: ");
+                inputString = getInputString();
+                inputString += '\0';
+                int deadlinePosition = inputString.indexOf("/");
+                int endPosition = inputString.indexOf("\0");
+                String taskName = inputString.substring(0, deadlinePosition);
+                String deadline = inputString.substring(deadlinePosition + 1, endPosition);
+                tasks.add(new Deadline(taskName, deadline));
+                tasks.get(counter).print();
+                counter++;
+                break;
+
+            case "event":
+                System.out.println("    _________________________________________");
+                System.out.println("    Please specify task: ");
+                inputString = getInputString();
+                inputString += '\0';
+                int deadlineStartPosition = inputString.indexOf("/");
+                int deadlineEndPosition = inputString.indexOf("|");
+                endPosition = inputString.indexOf("\0");
+                taskName = inputString.substring(0, deadlineStartPosition);
+                String deadlineStart = inputString.substring(deadlineStartPosition + 1, deadlineEndPosition);
+                String deadlineEnd = inputString.substring(deadlineEndPosition + 1, endPosition);
+                tasks.add(new Event(taskName, deadlineStart, deadlineEnd));
+                tasks.get(counter).print();
+                counter++;
+                break;
+
+            case "delete":
+                System.out.println("    Please specify task number: ");
+                taskNumber = getTaskNumber() - 1;
+                System.out.println("    _________________________________________");
+                tasks.get(taskNumber).printInList();
+                System.out.println("    _________________________________________");
+                System.out.println("    ");
+                tasks.remove(tasks.get(taskNumber));
+                counter--;
+                break;
+
+            default:
+                System.out.println("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+            break;
+        }
+    }
+
+        //When user types "bye"
+        System.out.println("    Bye. Hope to see you again soon!");
+        System.out.println("    _________________________________________");
+        System.out.println("     ");
+
+        PrintWriter fw = new PrintWriter("C:\\Users\\Manoj\\OneDrive - National University of Singapore\\Documents\\Module Files\\Y2S2\\CS2113\\iP\\src\\main\\java\\duke\\list.txt");
```

Comment:
1. particularly long line
2. using absolute path, that too reflecting a path on your machine makes this code unusable elsewhere
    3. best to use relative paths